lusSTR/wrappers/convert.py

```diff
@@ -102,6 +103,15 @@ def format_table(input, uas=False, kit="forenseq"):
             continue
 
         marker = STRMarkerObject(locus, sequence, uas=uas, kit=kit)
+        if locus == "D12S391" and kit == "powerseq":
+            if "." in str(marker.canonical):
+                check_sr += 1
+                if check_sr > 10:
+                    msg = (
+                        "Multiple microvariants identified at D12 locus. "
+                        "Please check STRait Razor version!!"
+                    )
+                    print(msg)
```

Comment:
The new SR config for D12 has a change of 9 bases, so using an older STRait Razor config file results in the assignment of almost all micro variants (e.g. ```13.1```, ```12.1```, ```9.1```, etc.). While these can occur when using the correct SR version, a large number of micro variants indicates a potential problem. This will just output a message directing the user to look into it (and identifies a potential fix).

Reply:
I don't want it to stop lusSTR though- just warn the user. Will printing to stderr do that?

Reply:
Nope. stderr is the conventional output stream for warning and error messages, stdout is for...output. 😀 Both are printed to the terminal, but can be redirected separately (e.g. `mycmd > out.txt 2> err.txt`). Printing warning messages to stderr is a best practice that makes sure that the user still sees them even if they redirect output to a file.

Reply:
Not super critical. You can handle this in #67—I didn't want it to hold up this PR.